docs/source/core_concepts/glossary.rst

```diff
@@ -259,12 +259,15 @@ Transaction Status Set
 ^^^^^^^^^^^^^^^^^^^^^^
 
  - NOT_RECEIVED: requested peer does not have this transaction.
- - MST_EXPIRED: this transactions is a part of MST pipeline and has expired.
+ - MST_PENDING: this transaction is a multisignature transaction which has to be signed by more keys (as requested in quorum field).
+ - MST_EXPIRED: this transaction is a multisignature transaction which is no longer valid and is going to be deleted by this peer from MST cache.
```

Comment:
Not sure about MST cache - client may not want to know anything about it. Maybe, explain in simpler terms or even leave as "is going to be deleted"?